amundsen_application/static/js/components/TableDetail/RequestMetadataForm/styles.scss

```diff
@@ -0,0 +1,48 @@
+@import 'variables';
+
+.request-component {
+  box-shadow: 0 0 24px -2px rgba(0, 0, 0, .2);
+  bottom: 75px;
+  position: fixed;
+  left: 25px;
+  z-index: 2;
+
+  &.expanded {
+    background-color: white;
+    border-radius: 0 0 6px 6px;
+    height: auto;
+    min-height: 450px;
+    padding: 32px;
+    width: 400px;
+    display: block;
```

Comment:
nit: Except `display:block` you could move all of these to the base `.request-component`.